psi4/driver/p4util/python_helpers.py

```diff
@@ -657,7 +657,23 @@ def _qcvar_reshape_get(key, val):
     """For QCVariables where the 2D psi4.core.Matrix shape is unnatural, convert to natural shape in ndarray."""
 
     reshaper = None
-    if key.upper().endswith("DIPOLE"):
+    if key.upper() == "MBIS_DIPOLES":
+        reshaper = (-1, 3)
+    elif key.upper() == "MBIS_QUADRUPOLES":
+        val = val.np.reshape(-1, 6)
+        val_new = []
+        for atom in range(len(val)):
+            val_new.append(_multipole_plumper(val[atom], 2))
+        val = np.array(val_new)
+        return val
+    elif key.upper() == "MBIS_OCTUPOLES":
+        val = val.np.reshape(-1, 10)
+        val_new = []
+        for atom in range(len(val)):
+            val_new.append(_multipole_plumper(val[atom], 3))
+        val = np.array(val_new)
```

Comment:
`val = np.array([_multipole_plumper(val[iat], 3) for iat in range(len(val))])`

list comprehensions are fun